dev/source/docs/building-setup-windows10.rst

```diff
@@ -28,13 +29,18 @@ Setup steps
    - Open the Microsoft Store and install Ubuntu 18.04 (Direct link: https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q)
    - Push the Launch button which will open a Bash terminal and ask for a username and password
 
+.. warning::
+
+    On Linux, the password is used often like for install new packages, don't lost it. Moreover, linux won't show you any caracters when you type your password.
```

Comment:
english correction, should be, "Becareful not to lose your Linux package because it is often required including when installing new packages.  Also Note linux will not display any characters as you type your password"